src/components/box/_box.scss

```diff
@@ -1,14 +1,19 @@
 $boxBorderColor: $graySecondary;
 $boxBorderRadius: 22px;
+$boxBorderSize: rhythm(1/8);
```

Comment:
All *vertical* dimensions should be relative to rhythm by default (with some exceptions, like `border-radius`).